src/Input/Select/Select.test.tsx

```diff
@@ -186,8 +186,11 @@ describe('when it is clicked', () => {
 
 describe('when an option is clicked', () => {
   it('should call the onOptionClick function', () => {
-    setupSelectOptionFromMenu();
+    const { randomOption } = setupSelectOptionFromMenu();
     expect(props.onOptionClick).toHaveBeenCalledTimes(1);
+    expect(props.onChange).toHaveBeenCalledWith(
+      randomOption.getAttribute('data-value')
```

Comment:
can read this from `randomOption.dataset.value`

Reply:
Done